src/stylesheets/components/_pagination.scss

```diff
@@ -137,12 +141,21 @@ $pagination-hover-token: nth($pagination-link-tokens, 2);
   border-color: transparent;
   color: color($text-color);
 
+  @media (forced-colors: active) {
+    outline: 2px solid buttonText;
+    color: buttonText;
+  }
+
```

Comment:
# Pagination

Before, the current page did not have depictive stylings so users could not easily see which page they are on, versus what is next or previous. This is mostly a problem in the unbounded variant where there are 4 page numbers grouped together.

Default:

![image](https://user-images.githubusercontent.com/25211650/154349579-75163f76-9d03-47ae-98b4-4768d3281565.png)

Unbounded:

![image](https://user-images.githubusercontent.com/25211650/154352226-7873af98-b064-4fdb-91c7-c64fc7a3cf46.png)



Current w/ high contrast:

![image](https://user-images.githubusercontent.com/25211650/154350426-75b7ad0d-4c11-4d4e-8809-965f158d0f93.png)
![image](https://user-images.githubusercontent.com/25211650/154352938-2197d8a9-7bd5-4deb-a1e1-b842dca2ebb4.png)


Fix w/ high contrast:

![image](https://user-images.githubusercontent.com/25211650/154350648-1380b223-614a-49d1-bac2-9540c8573c03.png)
![image](https://user-images.githubusercontent.com/25211650/154352978-312ea81f-2f78-4813-a99e-27c6e682e16f.png)